README.md

```diff
@@ -70,7 +70,7 @@ Here are some front-ends in various stages of development:
 
 * [xi-electron](https://github.com/acheronfail/xi-electron), a cross-platform front-end based on web-technologies.
 
-* [gxi](https://github.com/bvinc/gxi), a GTK+ front-end written in Rust.
+* [gxi](https://github.com/Cogitri/gxi), a GTK+ front-end written in Rust.
```

Comment:
I think it's fair to indicate that this is a fork, because the original project is dormant, and to include a link to the original project.

Reply:
Ah, sure. Would

`*  [gxi](https://github.com/Cogitri/gxi), a GTK+ front-end written in Rust - a fork of [gxi](https://github.com/bvinc/gxi)`

work for you?

Reply:
I would just say "forked from $link, which was abandoned".

Reply:
but I mean 🤷‍♂️ this is definitely unnecessarily nitty 😏

Reply:
Heh, no worries, please be as nitpicky as you want to! :)